geopandas/geoseries.py

```diff
@@ -388,42 +383,21 @@ def to_crs(self, crs=None, epsg=None):
         epsg : int
             EPSG code specifying output projection.
         """
-        from fiona.crs import from_epsg
```

Comment:
Can you update the docstring above as well? (something like that the `crs` argument takes a pyproj.CRS object or anything that accepted by `pyproj.CRS.from_user_input`

Reply:
Sure